src/test/java/sorinfoca/Tests/Tema7/MyTests.java

```diff
@@ -0,0 +1,94 @@
+package sorinfoca.Tests.Tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class MyTests extends TestConfig {
+
+    WebDriver driver;
```

Comment:
Please remove this line to avoid NullPointerException. We're going to use the driver from parent class. Thanks!